optimum/onnxruntime/configuration.py

```diff
@@ -715,11 +715,11 @@ class OptimizationConfig:
     disable_shape_inference: bool = False
 
     # ONNX Runtime 1.14.0 arguments
-    use_multi_head_attention = False
-    enable_gemm_fast_gelu_fusion = False
-    use_raw_attention_mask = False
-    disable_group_norm_fusion = True
-    disable_packed_kv = True
+    use_multi_head_attention: bool = False
+    enable_gemm_fast_gelu_fusion: bool = False
+    use_raw_attention_mask: bool = False
+    disable_group_norm_fusion: bool = False
+    disable_packed_kv: bool = False
```

Comment:
Reading the docstring of the class again, I think these two should be set to `True` by default (i.e. these optimizations are not done by default).